src/main/java/core/basesyntax/MyLinkedList.java

```diff
@@ -3,45 +3,171 @@
 import java.util.List;
 
 public class MyLinkedList<T> implements MyLinkedListInterface<T> {
+    private int size;
+    private Node<T> first;
+    private Node<T> last;
+
     @Override
     public void add(T value) {
+        linkLast(value);
     }
 
     @Override
     public void add(T value, int index) {
+        checkAddingIndex(index);
+        if (index == size) {
+            linkLast(value);
+        } else {
+            linkNode(value, getNode(index));
+        }
```

Comment:
With such change we may remove checkAddingIndex() method
```suggestion
        if (index == size) {
            linkLast(value);
        } else {
            checkIndex(index);
            linkNode(value, getNode(index));
        }
```